### PR TITLE
fix #36: WebUI not working on 64 bit only devices

### LINKED ISSUE #36: WebUI not working on 64 bit only devices
### Version

v1.0.0 (319-2e82988-release)

### Modules

my translation module with zygisk module version 1.0.0

### Description

Error when I open WebUI
Screenshot_2024-07-23-21-34-44-388_me bmax apatch

### Steps to reproduce

Open WebUI

### Logs

_No response_

### Confirmations

- [X] My environment meets the minimum requirements.
- [X] I have verified that this is not a duplicate issue.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by miuirussia**
## Why 

Write here why you think this should be merged

## Checkmarks

- [*] The modified functions have been tested.
- [*] Used the same indentation as the rest of the project.

### CODE REVIEW

**Comment by miuirussia** [comment]
@ThePedroo please, merge this fix

**Comment by RainyXeon** [comment]
@miuirussia He's not online rn số u have to be patient

File: webroot/js/main.js [hunk 1]
```diff
@@ -59,14 +59,20 @@ export function setErrorData(errorLog) {
   const ptrace64Cmd = await exec('/data/adb/modules/zygisksu/bin/zygisk-ptrace64 info')
 
   if (ptrace64Cmd.errno === 0) {
-    const lines = ptrace64Cmd.stdout.split('\n')
+    const [version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line] = ptrace64Cmd.stdout.split('\n')
```
Comment (ThePedroo) [inline webroot/js/main.js:62]:
```suggestion
    const [ version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line ] = ptrace64Cmd.stdout.split('\n')
```

File: webroot/js/main.js [hunk 2]
```diff
@@ -59,14 +59,20 @@ export function setErrorData(errorLog) {
...
 
     /* INFO: Root implementation and ReZygisk version parsing */
-    code_version.innerHTML = lines[0].split('Tracer ')[1].split('-')[0]
-    root_impl.innerHTML = lines[3].split(': ')[1]
+    if (daemon_running_line) {
+      code_version.innerHTML = version_line.split('Tracer ')[1].split('-')[0]
+      root_impl.innerHTML = daemon_running_or_root_impl_line.split(': ')[1];
```
Comment (ThePedroo) [inline webroot/js/main.js:67]:
```suggestion
      root_impl.innerHTML = daemon_running_or_root_impl_line.split(': ')[1]
```

File: webroot/js/main.js [hunk 3]
```diff
@@ -84,14 +90,20 @@ export function setErrorData(errorLog) {
   const ptrace32Cmd = await exec('/data/adb/modules/zygisksu/bin/zygisk-ptrace32 info')
 
   if (ptrace32Cmd.errno === 0) {
-    const lines = ptrace32Cmd.stdout.split('\n')
+    const [version_line, _, process_line, daemon_running_or_root_impl_line, daemon_running_line] = ptrace32Cmd.stdout.split('\n')
```
Comment (ThePedroo) [inline webroot/js/main.js:93]: Ditto

File: webroot/js/main.js [hunk 4]
```diff
@@ -84,14 +90,20 @@ export function setErrorData(errorLog) {
...
 
     /* INFO: Root implementation and ReZygisk version parsing -- Necessary if 64-bit fails */
-    code_version.innerHTML = lines[0].split('Tracer ')[1].split('-')[0]
-    root_impl.innerHTML = lines[3].split(': ')[1]
+    if (daemon_running_line) {
+      code_version.innerHTML = version_line.split('Tracer ')[1].split('-')[0]
+      root_impl.innerHTML = daemon_running_or_root_impl_line.split(': ')[1];
```
Comment (ThePedroo) [inline webroot/js/main.js:98]: Ditto

**Review by ThePedroo — CHANGES_REQUESTED** [review]

**Review by ThePedroo — APPROVED** [review]
LGTM

**Pull request merged**